utility/src/main/java/org/oppia/android/util/parser/CustomBulletSpan.kt

```diff
@@ -18,22 +20,28 @@ import org.oppia.android.util.R
  * Reference: https://github.com/davidbilik/bullet-span-sample
  */
 class CustomBulletSpan(context: Context) : LeadingMarginSpan {
-  private var bulletRadius: Int = 0
-  private var gapWidth: Int = 0
-  private var yOffset: Int = 0
-  private var bulletLeadingMargin: Int = 0
+  private val bulletRadius: Int
+  private val gapWidth: Int
+  private val yOffset: Int
+  /** The space between the start of the line and the bullet. */
+  private val spacingBeforeBullet: Int
+  /** The space between the bullet and the text. */
+  private val spacingBeforeText: Int
+  /** The total spacing between the start of the line and the text. */
+  private val totalSpacingToTextStart: Int
+  private var bulletPath: Path? = null
 
   init {
     bulletRadius = context.resources.getDimensionPixelSize(R.dimen.bullet_radius)
     gapWidth = context.resources.getDimensionPixelSize(R.dimen.bullet_gap_width)
     yOffset = context.resources.getDimensionPixelSize(R.dimen.bullet_y_offset)
-    bulletLeadingMargin = context.resources.getDimensionPixelSize(R.dimen.bullet_leading_margin)
+    spacingBeforeBullet = context.resources.getDimensionPixelSize(R.dimen.spacing_before_bullet)
+    spacingBeforeText = context.resources.getDimensionPixelSize(R.dimen.spacing_before_text)
+    totalSpacingToTextStart = spacingBeforeBullet + spacingBeforeText + bulletRadius * 2
```

Comment:
All these can be done as joint description above.

```
private val bulletRadius: Int = context.resources.getDimensionPixelSize(R.dimen.bullet_radius)
  private val gapWidth: Int = context.resources.getDimensionPixelSize(R.dimen.bullet_gap_width)
  private val yOffset: Int = context.resources.getDimensionPixelSize(R.dimen.bullet_y_offset)

  /** The space between the start of the line and the bullet. */
  private val spacingBeforeBullet: Int = context.resources.getDimensionPixelSize(R.dimen.spacing_before_bullet)

  /** The space between the bullet and the text. */
  private val spacingBeforeText: Int = context.resources.getDimensionPixelSize(R.dimen.spacing_before_text)

```

Reply:
@rt4914 do I have to remove the init block?

Reply:
If possible yes but not compulsory.

Reply:
@rt4914 if initialize them at the declaration then i might have to add `ktlint avoid max line limit`
![image](https://user-images.githubusercontent.com/54740946/103271390-23611900-49e0-11eb-868d-685fa54d9277.png)

Reply:
You can do something like this:
`val resources = context.resources`. And you can use `resources`.

Also, you can remove `: Int` too.

Reply:
Done